.github/workflows/ci.yml

```diff
@@ -578,7 +578,42 @@ jobs:
             bash -euo pipefail -c \
             "./build_tools/cmake/build_riscv.sh && tests/riscv32/smoke.sh"
 
-  riscv64:
+  linux_riscv32:
```

Comment:
It looks like `linux_riscv32` and `linux_riscv64` are almost identical. Wondering if we should move to the matrix approach here (https://github.com/iree-org/iree/blob/main/.github/workflows/benchmarks.yml#L93)

Reply:
Makes sense. Let me work on that. Thanks for the suggestion.